packages/framework/src/servers/express.ts

```diff
@@ -5,6 +5,20 @@ import { NovuRequestHandler, ServeHandlerOptions } from '../handler';
 import { Either } from '../types';
 import { type SupportedFrameworkName } from '../types';
 
+/*
+ * Re-export all top level exports from the main package.
+ * This results in better DX reduces the changes of the dual package hazard for ESM + CJS packages.
```

Comment:
Great callout.

Small typo:
```suggestion
 * This results in better DX reduces the chances of the dual package hazard for ESM + CJS packages.
```